build/patch-charts.sh

```diff
@@ -155,10 +154,12 @@ function patchGalley() {
 
   convertClusterRoleBinding "${HELM_DIR}/istio-control/istio-discovery/templates/clusterrolebinding.yaml" ""
   sed_wrap -i -e '/- "discovery"/ a\
+{{- if not .Values.global.clusterScoped }}\
           - --memberRollName=default\
+          - --disableNodeAccess=true\
+{{- end}}\
           - --enableCRDScan=false\
```

Comment:
we don't need to disable this anymore in cluster-scoped mode, do we?

Reply:
I wasn't sure, but I've now enable this for cluster-wide installs.

Reply:
Could you explain why?